### ISSUE TITLE
Nginx config file is broken if --chain-path is missing

### ISSUE THREAD

**Issue by juniorz** [twstrike/letsencrypt#3]
Scenario:

``` gherkin
Given I already have a LE certificate
And I want to deploy it to a new nginx installation
When I run `letsencrypt install`
Then I should have all nginx files changed
And nginx should be accessible via HTTPS
```

This happens when I try to install a certificate previously obtained using `letsencrypt certonly`.

versions

``` console
root@dev:/home/letsencrypt/client# cat /etc/issue
Ubuntu 15.10 \n \l
root@dev:/home/letsencrypt/client# dpkg -s nginx | grep 'Version'
Version: 1.9.3-1ubuntu1
```

command

``` console
letsencrypt install --installer nginx \
  -d letsencrypt.you \
  --cert-path /etc/letsencrypt/live/letsencrypt.you/cert.pem \
  --key-path /etc/letsencrypt/live/letsencrypt.you/privkey.pem \
  --text --debug -vvvvvvv
```

nginx config problem:

``` console
root@dev:/home/letsencrypt/client# nginx -t
nginx: [emerg] invalid number of arguments in "ssl_trusted_certificate" directive in /etc/nginx/nginx.conf:13
nginx: configuration file /etc/nginx/nginx.conf test failed
```

nginx config file diff (from default config files that ship with the package)

``` udiff
diff --git a/nginx.conf b/nginx.conf
index 01a4a21..b8e071f 100644
--- a/nginx.conf
+++ b/nginx.conf
@@ -6,9 +5,20 @@ events {
     worker_connections 768;
     # multi_accept on;
 }
-
 http {

+    server {
+        ssl_stapling_verify on;
+        ssl_stapling on;
+        ssl_trusted_certificate;
+        include /etc/letsencrypt/options-ssl-nginx.conf;
+        ssl_certificate_key /etc/letsencrypt/live/letsencrypt.you/privkey.pem;
+        ssl_certificate;
+        error_log /var/lib/letsencrypt/error.log;
+        access_log /var/lib/letsencrypt/access.log;
+        listen 443 ssl;
+        server_name letsencrypt.you;
+    }
     ##
     # Basic Settings
     ##
@@ -29,10 +35,9 @@ http {
     ##
     # SSL Settings
     ##
-
-       ssl_protocols TLSv1 TLSv1.1 TLSv1.2; # Dropping SSLv3, ref: POODLE
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    # Dropping SSLv3, ref: POODLE
     ssl_prefer_server_ciphers on;
-
     ##
     # Logging Settings
     ##
```

**Comment by juniorz** [comment]
The problem seems to be `letsencrypt install` requires `--cert-path` and `--key-path` but the nginx installer also requires `--chain-path` (which is missing my command invocation).

Expected behavior: the nginx plugin should fail if `--chain-path` is not provided OR should not set `ssl_trusted_certificate` in the config file.

I'm not sure about the implication of a missing `ssl_trusted_certificate` in the configuration file, and whether this is expected.

The display of mandatory `--cert-path` and `--key-path` were addressed as part of https://github.com/letsencrypt/letsencrypt/issues/996. There is still the question if should `--chain-path` be mandatory.

**Comment by juniorz** [comment]
This configuration is used to enable OCSP stapling on nginx >= 1.3.7. It seems to be used in the revocation process, so I think it should require `--chain-path` in this case rather than not enabling OCSP stapling.

I'm going to abort the installation in this case.

**Comment by juniorz** [comment]
Note: both `chain_path` and `fullchain_path` default to `None` in apache plugin, so I assume they are optional by design.

**Comment by juniorz** [comment]
Fixed by: 4ac8fd18517bb712d4cf5b3698b47a6e1ed691d8

(I'm not adding references to OUR issues on the commits to don't mess up with the upstream repo when they merge the changes)
